pkgs/top-level/all-packages.nix

```diff
@@ -6543,11 +6543,13 @@ with pkgs;
 
   mkdocs = with python3Packages; toPythonApplication mkdocs;
 
+  mold = lib.warn "mold package will switch from being unwrapped to being wrapped; use explicit mold-wrapped or mold-unwrapped in the interim" mold-wrapped;
```

Comment:
Warnings are errors in CI, might be better to just leave the warning out.

Reply:
CI is still failing. It's not just the usage but declaration that'll cause CI to fail.

Reply:
Yeah, I think we should prefer `mold` over `mold-wrapped` but `mold` should be the wrapped package while `mold-unwrapped` is the unwrapped package.

Reply:
We could have `lib.warn` in `aliases.nix` but not in `all-packages.nix`.